NERODevelopment/src/controllers/offviewcontroller.cpp

```diff
@@ -0,0 +1,53 @@
+#include "offviewcontroller.h"
+
+OffViewController::OffViewController(Model *model, QObject *parent)
+    : QObject{parent}, m_packTemp(0.0), m_motorTemp(0.0), m_stateOfCharge(0.0) {
+    this->m_model = model;
+    connect(m_model, &Model::updateAttributesFromModel, this,
```

Comment:
this compiles? i dont think this function exists in the model, you want the models currentDataDidChange Signal